_scripts/app2-4AltAnswer.R

```diff
@@ -0,0 +1,66 @@
+{
+
+  # Application 2-4.R ----
+  
+  ## This is an extension to app2-4.R employing 'if else' statements
+  
+  rm(list=ls());
+  options(show.error.locations = TRUE);
+  
+  ## Initialize vector and vector length
+  
+  data <- c(94, 102, 89, 105, 78, 85);
+  lengthData <- length(data);
+  
+  ## Boolean variables (2.1 - 2.4)
+  
+  greaterThan100 <- FALSE; 
+  lessThan70 <- FALSE;
+  equalTo78 <- FALSE;
+  equalTo87 <- FALSE;
+  
+  ## State variables
+  
+  dataMin <- data[1]; # Assign to first value in 'data'
+  dataMax <- data[1]; # Assign to first value in 'data'
+  dataSum <- 0; # Assign to 0
+  
+  ## Loop through each value (j) in 'data' vector
+  
+  for(j in 1:lengthData){
+    
```

Comment:
Right now you have 6 codeblocks and only one can be executed because everything is in an if-else structure.  But, I would argue the first two codeblocks are mutually exclusive and the last four mutually exclusive, but all six codeblocks are not mutually exclusive (e.g., you could have a minimum value equal to 78 or a max value greater than 100)